blueprints/ember-cli-eslint/files/.eslintrc.js

```diff
@@ -1,3 +1,7 @@
+var path = require('path');
+
 module.exports = {
-  extends: './node_modules/ember-cli-eslint/coding-standard/ember-application.js'
+  extends: [
+    path.join(__dirname, 'node_modules/ember-cli-eslint/coding-standard/ember-application.js')
```

Comment:
IMO, we should never assume a given file is nested inside node_modules. We should use require.resolve (I wasn't talking about path.resolve above).